packages/client/hooks/useEditorState.ts

```diff
@@ -3,10 +3,11 @@ import {convertFromRaw, EditorState} from 'draft-js'
 import {useEffect, useRef, useState} from 'react'
 import makeEditorState from '../utils/draftjs/makeEditorState'
 import mergeServerContent from '../utils/mergeServerContent'
+import remountDecorators from '../utils/draftjs/remountDecorators'
 
-const useEditorState = (content?: string | null | undefined) => {
+const useEditorState = (content?: string | null | undefined, searchQuery?: string | null) => {
```

Comment:
Ideally, I think `useEditorState` shouldn't need to know about the `spotlightSearchQuery`. I'm not too familiar with DraftJS, and I've always found it to be a pain when dealing with it, but I think we can set state with the decorator. See here: https://draftjs.org/docs/advanced-topics-decorators/#setting-new-decorators